install-dev/models/install.php

```diff
@@ -114,21 +116,14 @@ public function generateSettingsFile(
                     'database_engine' =>  $database_engine,
                     'cookie_key' => $cookie_key,
                     'cookie_iv' =>  $cookie_iv,
+                    'new_cookie_key' => $key->saveToAsciiSafeString(),
```

Comment:
Yeah, it's used at least for customer passwords, and probably in many modules. I think it's very adventurous (suicidal?) to rename it!

What about `THE_REAL_COOKIE_KEY_NOT_THE_UNIVERSAL_SALT`?

To be honest, I think `cookie_key_2` is better because we can have a `cookie_key_x` while having a `new_new_cookie_key` will be annoying.
Also, in 3 years, we will probably rely on it, but is it actually considered *new*? :D